_posts/2018-09-07-redis-ruby-bloom-filter.md

```diff
@@ -0,0 +1,355 @@
+---
+layout: post
+title: "Implementing a custom Redis and in-memory bloom filter"
+date: 2018-09-05 12:00:00 -0800
+cover: /assets/images/bloom_filter.svg
+excerpt: We recently changed our bloom filter implementation by using a custom Redis and an in-memory bloom filter written in Ruby. We will go through few iterations at solving a real problem writing a custom bloom filter from scratch.
+options:
+  - full-bleed-cover
+authors:
+  - name: Dalibor Nasevic
+    url: https://dalibornasevic.com
+    photo: https://avatars.githubusercontent.com/dalibor
+---
+
+
+Recently, we had to do an optimization that involved replacing a bloom filter built in-memory and stored on Amazon S3 to a combination of a Redis bloom filter and an in-memory bloom filter. In this blog post we'll go through the reasoning about this change as well as the details of the bloom filter implementation we landed on. Let's first start with a very brief info about what is a bloom filter.
```

Comment:
I wonder if this post should start out with a blurb describing your product? A reader might think "we" means all of GoDaddy

Reply:
Also, it would be interesting

Reply:
`very brief info about what is a bloom filter` could be phrased better. I'd suggest something like `brief introduction to bloom filters`